cmd/crictl/image.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package main
 
 import (
+	"encoding/json"
```

Comment:
should this be added to the `go.mod`?

Reply:
no, this is a go standard library package